malboxes/profiles/win10_64_analyst.json

```diff
@@ -21,7 +22,11 @@
 
 		"output_directory": "builds",
 
-		"iso_url": "http://care.dlservice.microsoft.com/dl/download/C/3/9/C399EEA8-135D-4207-92C9-6AAB3259F6EF/10240.16384.150709-1700.TH1_CLIENTENTERPRISEEVAL_OEMRET_X64FRE_EN-US.ISO",
+
+		"iso_urls": [
+			"file://{{ user `iso_path` }}/10240.16384.150709-1700.TH1_CLIENTENTERPRISEEVAL_OEMRET_X64FRE_EN-US.ISO",
```

Comment:
Wouldn't it be better to have the iso filename configurable as well instead of a hardcoded one ?